tasks/main.yml

```diff
@@ -10,3 +10,41 @@
   package:
     name: cockpit-session-recording
   when: install_session_player
+
+- name: configure basic sssd
```

Comment:
Your task name and the actual task performed don't match for this task and the next (they are swapped).